### ISSUE TITLE
Number of transactions per day is always equal to number of total users!

### ISSUE THREAD

**Issue by mostaphaRoudsari** [mostaphaRoudsari/visualization-experiments#6]
@JonatanS this looks like a bug to me. Are you sure that you are giving me number of transactions per office in each time step and  not number of users per office. If you check in data for each time step

d.new_users + d.existing_users = sum(d.usage_data.values())

looks like a bug to me!

**Comment by mostaphaRoudsari** [comment]
also you can check it here visually, hover on the stacked graphs...
http://mostapharoudsari.github.io/visualization-experiments/TTXMap

**Comment by JonatanS** [comment]
@mostaphaRoudsari yes you are right. I'll change it... Still working on the AWS stuff today

**Comment by mostaphaRoudsari** [comment]
@JonatanS thanks. Can't wait to see how it will look with real data. It looks like Tetris right now. I made so many modifications to the code. It looks much better now and you get stacked graph also for each office and a text report...
